### PR TITLE
 Identify persona for SEED onboarding, Included persona based onboarding (PO and vendors), Included Step onboarding, Replaced the old TP support URL with the new url and Updated Onboarding FAQs for Step onboarding

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
1. Identify persona for SEED onboarding
 2.  Included persona-based onboarding (PO and vendors), 
 3. Included Step onboarding, 
 4. Replaced the old TP support URL with the new url
 5. Updated Onboarding FAQs for Step onboarding

### CODE REVIEW

File: onboard-device/windows.md [hunk 1]
```diff
@@ -86,20 +61,67 @@ foreach ($name in $enrollmentsKey.GetSubKeyNames()) {
...
+
+| Status | Description | Action required |
+|---| ---| ---|
+| **triggered, waiting for software installation (step 1 of 2)**| Your SEED onboarding has been triggered on the device and is waiting for the software installation to be completed. | When you click the refresh button after a successful software installation, the status changes to **software installed, waiting for backend onboarding**.|
+| **software installed, waiting for backend onboarding (step 2 of 2)**| Required software has been installed on the device and waiting for backend onboarding.  | When you click the refresh button after a successful backend onboarding, the status changes to **onboarded** . |
+| **onboarded** | Your SEED onboarding is successful. | Go to step 8 in this section.  |
+| **failed(*Reason for failure*)** | Your SEED onboarding failed due to the  error mentioned within the parentheses. | Action required to resolve this failure is generally mentioned in the parentheses. Complete the suggested action required by you. | 
+
+
+8. Check your inbox (organisational email address) to see if you have received the successfully onboarded email.
```
Comment (cricketer94) [inline onboard-device/windows.md:104]: @SenthilKumarSivakami shouldn't we get them to check whether they received a failed email status first first?

File: overview.md [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+
+This is the Identity Access Management(IAM) and Single Sign-On(SSO) solution for accessing SGTS and GCC services.
+
+### **Cloudflare**
+
+The security platform that enforces Zero Trust network access allowing faster and safer connections to the Internet and applications. This comprises of the following:<br>- **Cloudflare WARP**: Replaces the traditional VPN clients.<br>- **Cloudflare Gateway**: Blocks and protects from malicious content.<br>- **Cloudflare Access**: Evaluates every request for user identity and device context.
+
+### **DEEP**
+
+Device management layer of SEED. It establishes a robust security baseline automatically​ and prevents insecure or compromised devices from accessing engineering resources.​ DEEP manages the following:<br>- **Microsoft Intune**: Provides device and application management including remote application deployment and selective device wipe.<br>- **Microsoft Defender Advanced Threat Prevention**: Enterprise class vulnerability management, threat detection and response security solution.<br>- **Tanium**: Works with Cloudflare to ensure posture-based conditional access to the endpoint assets.
```
Comment (cricketer94) [inline overview.md:52]: nit: Not important, we can edit later, but I'd like to edit this portion slightly to mention the management of the various automations that work behind the scenes to ensure the EDR portion of our platform works.

**Review by cricketer94 — COMMENTED** [review]
@SenthilKumarSivakami I only just wondering whether we need to indicate things about the failed onboarding email? If not a big deal this can be pushed out

File: images/macos-get-intune-device-id-new.png
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply images/macos-get-intune-device-id-new.png:1]: No, @eu-tee  only wanted to blur the user name and email . Is it a security issue to display the Intune device ID like this..then I will blur it :-)

File: onboard-device/windows.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply onboard-device/windows.md:104]: I was not aware of this, the page https://confluence.ship.gov.sg/display/SEED/SEED+Onboarding+for+PO did not mention that the user has to check the failed onboarding email. so I did not update it. let me see where to fit it.

Comment (SenthilKumarSivakami) [reply onboard-device/windows.md]: I have updated both mac and windows as needed.

File: overview.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply overview.md:52]: noted will do in the next cycle. Thanks!

File: onboard-device/mac-os.md [hunk 3]
```diff
@@ -131,15 +66,95 @@ echo "$actual_id"
...
+
+6. Ensure the device you are onboarding is connected to the Internet so that Intune is able to install the required software and configurations.
+
+7. After 30-60 minutes, check your inbox (organisational email address) to see if you have received any email regarding your onboarding status.
+
+8. Choose the appropriate step:
+
+   a. If you have received a successfully onboarded email, skip rest of the steps in this section and proceed to [Step 3: Verify installation](#step-3-verify-installation).
+
+   b. If you have **not yet received** the **successfully onboarded email** or if you **have received** a **failed onboarding email**, complete the following step on [TechPass portal](https://portal.techpass.gov.sg/).
```
Comment (cricketer94) [inline onboard-device/mac-os.md:103]: Just add that TechPass portal should be accessed on the non-SE GSIB

**Review by eu-tee — APPROVED** [review]
Ok for now, but follow up with edits asap.

For Step 8, when you mention them to go TechPass portal, remind them that they need to access it on their non-SE GSIB.

Change the Tunein id to dummy string.

**Review by cricketer94 — APPROVED** [review]
Approved.

**Pull request merged**